### ISSUE TITLE
DualSense ang Gyro/Tilt

### ISSUE THREAD

**Issue by Amnesia1000** [mgba-emu/mgba#2525]
Dualsense's motion sensor is not recognized by mGBA on linux.

As a clarification, the motion sensor is recognized in emulators like Yuzu and citra without any problem, and in cemu through a udp recognition script.

I hope (if it's not my mistake when configuring something wrong) they can make dualsense compatible with the emulator, since the functions make it a great command for emulators.

Tested on the latest dev version of the emulator with "Warioware Twisted", "Yoshi Topsy Turvy" and "Kirby tilt and tumble" games.

**Comment by endrift** [comment]
This should already be compatible in the dev builds. If it's not, something went wrong. What OS?

**Comment by Amnesia1000** [comment]
I think I was able to solve it, I just realized that through the web they also offer an appimage file. I downloaded it and it worked, apparently the difference is the libedit library. In manjaro the latest version is 0.65 (libedit.so.0.56) but the appimage version asks for version 2 (libedit.so.2). Copy the library from the lib folder of the appimage to the system and now it takes the gyroscope without any problem.
I leave the error open just to have confirmation if it could be this, or something else.
